addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml

```diff
@@ -21,7 +21,10 @@
 				<options>
 					<option value="0">North America (915MHz)</option>
 					<option value="1">Europe (868MHz)</option>
-					<option value="2">Australia (915MHz)</option>
+					<option value="2">Australia</option>
+					<option value="3">Fine Offset (915MHz)</option>
```

Comment:
I think if you owned the device itself it probably makes more sense.  The device documentation says this:

"m0“ selects the USA band at 915 MHz, „m1“ selects the Europe band at 868 MHz and “m2” selects Australia frequency band. “m3” selects reception of Fine Offset WH24 sensor unit in 915 MHz band and “m4” in 868 MHz band.

I've added the un-documented NZ mode ("m5") but otherwise tried to stay consistent with the original device documentation, including removing the Aus frequency.

The issue is that I don't think this parameter is really just band.  I think that the Davis devices that this thing supports have both bands and frequency hopping protocols that depend on local frequency regulations.  So in Aus it might be in 915mhz but the frequency hopping will be different from NA.   Then it seems to support a specific device (Fine Offset) that works in the same bands on different protocols. 

I don't *know* this for sure though so thought best to stick to what the manufacturer says, even though it is a bit confusing.